homeassistant/components/braviatv/__init__.py

```diff
@@ -44,3 +67,237 @@ async def async_unload_entry(hass, config_entry):
 async def update_listener(hass, config_entry):
     """Handle options update."""
     await hass.config_entries.async_reload(config_entry.entry_id)
+
+
+class BraviaTVClient(DataUpdateCoordinator[None]):
```

Comment:
Let's call this coordinator.
```suggestion
class BraviaTVCoordinator(DataUpdateCoordinator[None]):
```